docs/criufeature.md

```diff
@@ -0,0 +1,93 @@
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# CRIU feature
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+:fontawesome-solid-pencil:{: .note aria-hidden="true"} **Note:** CRIU support is a technical preview feature that is intended for container environments. All APIs and options are subject to change. CRIU support requires the version of CRIU that is a package in Semeru UBI container images.
+
+In Eclipse OpenJ9&trade;, Checkpoint Restore in User Space (CRIU) feature is available as an API that can be used to stop the VM at a particular checkpoint, save the state, and then enable the VM to resume from the point where it was stopped. The time between the VM start and checkpoint is referred to as the checkpoint phase. The checkpointed state is written to one or more image files with the memory content available during the checkpoint phase, such as memory pages, methods, file systems, processes, network connections, and security components. You can restore the checkpoint image from these files in as many environments as possible with improved start-up time and hence with better VM performance.
+
+For more information about the CRIU feature and OpenJ9 CRIU support, see the following blog posts:
+
+- [Fast JVM startup with OpenJ9 CRIU Support](https://blog.openj9.org/2022/09/26/fast-jvm-startup-with-openj9-criu-support/)
+- [Getting started with OpenJ9 CRIU Support](https://blog.openj9.org/2022/09/26/getting-started-with-openj9-criu-support/)
+- [OpenJ9 CRIU Support: A look under the hood](https://blog.openj9.org/2022/10/14/openj9-criu-support-a-look-under-the-hood/)
+- [OpenJ9 CRIU Support: A look under the hood (part II)](https://blog.openj9.org/2022/10/14/openj9-criu-support-a-look-under-the-hood-part-ii/)
+
+## Using the CRIU feature
+
+CRIU feature is not enabled by default, you must explicitly invoke it. You can invoke the feature with the `-XX:+EnableCRIUSupport` option. When thus enabled, the VM starts with support for checkpoint and restore. The VM will launch the JIT, GC, Security, and other internal components that make checkpoint and restore feasible.
+
+For more information about the `-XX:+EnableCRIUSupport` option, see the [`-XX:[+|-]EnableCRIUSupport`](xxenablecriusupport.md) topic.
+
+## Configuring the CRIU support feature
+
+You can use command line options to configure the CRIU support to use CRIU capabilities that are provided by the operating system as well as VM support.
+
+### New options for checkpoint and restore process
+
+- [`-XX:[+|-]EnableCRIUSupport`](xxenablecriusupport.md) - Enables starting of the VM with support for checkpoint and restore.
+- [`-XX:[+|-]CRIURestoreNonPortableMode`](xxcriurestorenonportablemode.md) - Indicates to the VM that only a single checkpoint can be taken in the VM lifetime.
```

Comment:
You could just use "Specifies" instead of "Indicates to the VM" all VM options are indicating something to the VM. I think the summary sentence needs to relate to the option name otherwise it's rather confusing.